src/features/timeline/TimelineIndexPage.js

```diff
@@ -49,7 +47,7 @@ const CycleIndexPage = ({ entryId }) => {
           <DatePicker date={selectedDate} />
           <IconButton
             aria-label="Scroll to today"
-            onClick={() => navigate(TIMELINE.to)}
+            onClick={() => navigate(`/timeline/${entryIdFromDate(new Date())}`)}
```

Comment:
**suggestion: (non-blocking)** I think you can skip "/timeline/" and it will add it to the end of current path, makes sense in this use case I think...

Reply:
Up to you but i sometimes prefer to be more explicit so future sara does not get confused

Reply:
Makes sense, you do you

Reply:
Merged and closed the other one :D